lib/bootloader_setup.pm

```diff
@@ -160,13 +160,17 @@ sub bootmenu_default_params {
         if (!get_var("NICEVIDEO")) {
             if (is_jeos) {
                 type_string_very_slow "console=$serialdev ";    # to get crash dumps as text
-                type_string_very_slow "console=tty ";           # to get crash dumps as text
+
+                #See bsc#1011815, last console set as boot parameter is linked to /dev/console and doesn't work if set to serial device
+                type_string_very_slow "console=tty ";
             }
             else {
                 type_string_very_slow "plymouth.ignore-serial-consoles ";    # make plymouth go graphical
                 type_string_very_slow "linuxrc.log=$serialdev ";             # to get linuxrc logs in serial
                 type_string_very_slow "console=$serialdev ";                 # to get crash dumps as text
-                type_string_very_slow "console=tty ";                        # to get crash dumps as text
+
+                #See bsc#1011815, last console set as boot parameter is linked to /dev/console and doesn't work if set to serial device
+                type_string_very_slow "console=tty ";
```

Comment:
instead of copy/paste better extract a function for the common parts here and above, e.g. `type_console_parameter' and call this sub in both cases. Then the comment can be there as well.